src/test/java/com/aws/greengrass/lifecyclemanager/LogManagerHelperTest.java

```diff
@@ -67,8 +77,6 @@
 class LogManagerHelperTest {
     @TempDir
     protected Path tempRootDir;
-    @TempDir
```

Comment:
`@TempDir` just injects the same directory if you use multiple of them. There is an [ongoing feature request](https://github.com/junit-team/junit5/issues/1967) for Junit.